doc/upgrading.md

```diff
@@ -52,6 +52,17 @@ enable generating clients:
             </configuration>
 ```
 
+Add FOLIO Maven repository for plugins after existing `<repositories>` section:
+```xml
+  <pluginRepositories>
+    <pluginRepository>
+      <id>folio-nexus</id>
+      <name>FOLIO Maven repository</name>
+      <url>https://repository.folio.org/repository/maven-folio</url>
+    </pluginRepository>
+  </pluginRepositories>
+```
+
```

Comment:
This needs to be added to the README as well: https://github.com/folio-org/raml-module-builder#step-2-include-the-jars-in-your-project-pomxml

Reply:
Ok. Will do